.github/workflows/stale.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   stale:
-
+    if: ${{ github.repository == 'kubeflow/pipelines' }}
```

Comment:
Why do we need this? Do we want to prevent forks from using this workflow?

Reply:
Yes